packages/frontend/src/components/Teams.tsx

```diff
@@ -1,12 +1,14 @@
-import useFetch from '../hooks/useFetch';
+// import { useFetch} from '../hooks/useFetch';
+import { useCurrentSeason } from '../hooks/useCurrentSeason';
 import useTeams from '../hooks/useTeams';
 import { HighScoreTable } from './HighScoreTable';
 import { MovingBot } from './MovingBot';
 import { Table } from './Table';
 
 export const Teams = () => {
   const teams = useTeams();
-  const { response: currentSeason } = useFetch('api/seasons/current', '0');
+  const currentSeason = useCurrentSeason();
```

Comment:
what is returned by this hook?

Reply:
the hook returns the fetched currentSeason state, defined by the ISeasonDto interface.
it seems to be redundant and should probably be removed.